packages/atomic/src/pages/examples/insights.html

```diff
@@ -116,9 +116,17 @@
                   }
                 </style>
                 <atomic-insight-result-action-bar>
-                  <atomic-insight-result-action tooltip="test"></atomic-insight-result-action>
-                  <atomic-insight-result-action tooltip="test"></atomic-insight-result-action>
-                  <atomic-insight-result-action tooltip="test"></atomic-insight-result-action>
+                  <atomic-insight-result-action
+                    tooltip="copy to clipboard"
+                    action="copyToClipboard"
+                    icon='<svg width="26" height="26" viewBox="0 0 26 26" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M22 1H9C7.9 1 7 1.9 7 3V4H19C20.1 4 21 4.9 21 6V20H22C23.1 20 24 19.1 24 18V3C24 1.9 23.1 1 22 1ZM19 8C19 6.9 18.1 6 17 6H4C2.9 6 2 6.9 2 8V23C2 24.1 2.9 25 4 25H17C18.1 25 19 24.1 19 23V8ZM10 11.5C10 11.8 9.8 12 9.5 12H5.5C5.2 12 5 11.8 5 11.5V10.5C5 10.2 5.2 10 5.5 10H9.5C9.8 10 10 10.2 10 10.5V11.5ZM14 19.5C14 19.8 13.8 20 13.5 20H5.5C5.2 20 5 19.8 5 19.5V18.5C5 18.2 5.2 18 5.5 18H13.5C13.8 18 14 18.2 14 18.5V19.5ZM16 15.5C16 15.8 15.8 16 15.5 16H5.5C5.2 16 5 15.8 5 15.5V14.5C5 14.2 5.2 14 5.5 14H15.5C15.8 14 16 14.2 16 14.5V15.5Z"/></svg>'
```

Comment:
That's so so for an example 🤷

If we tighten up the available actions to be some known list instead of any string, the icon could be inferred internally in the component, without requiring implementers to provide those.

Reply:
I don't like the fact that the icon is a straight up svg data nonplus.. :(